test/Models/mobilenet-without-batchnorm.mlir

```diff
@@ -0,0 +1,900 @@
+// RUN: tpp-opt %s -decompose-conv-to-matmul-or-brgemm -empty-tensor-to-alloc-tensor \
```

Comment:
Try to use the default pipeline option. We should be moving to that for all integration tests soon enough.

Reply:
FYI, at the moment `-expand-strided-metadata -lower-affine` have to be added separately after the default pipeline.
I'll add these two passes to `tpp-run` soon.